File: src/NubipApp.Models/Domain/AuditoriumType.cs
using System.Collections.Generic;

namespace NubipApp.Models.Domain
{
    public class AuditoriumType
    {
        public int AuditoriumTypeId { get; set; }
        public string Title { get; set; }
        public bool Computer { get; set; }
        
        public ICollection<Auditorium> Auditoriums { get; set; }
    }
}

File: src/models/auditoriumType.ts
import mongoose from "mongoose";

let Schema = mongoose.Schema;

let AuditoriumTypeSchema = new Schema({
	auditoriumTypeId: Number,
	title: String,
	computer: Boolean
});

let AuditoriumType = mongoose.model('AuditoriumType', AuditoriumTypeSchema);

export default AuditoriumType;